data/mods/Xedra_Evolved/mod_interactions/Defense_Mode/overmap_specials.json

```diff
@@ -0,0 +1,654 @@
+[
+  {
+    "type": "overmap_special",
+    "id": "defense_mode_field_office",
+    "overmaps": [
+      { "point": [ 1, 1, 0 ], "overmap": "dm_field_office_5_south" },
+      { "point": [ 0, 1, 0 ], "overmap": "dm_field_office_6_south" },
+      { "point": [ 1, 0, 0 ], "overmap": "dm_field_office_7_south" },
+      { "point": [ 0, 0, 0 ], "overmap": "dm_field_office_8_south" },
+      { "point": [ 1, 1, -1 ], "overmap": "dm_field_office_1_south" },
+      { "point": [ 0, 1, -1 ], "overmap": "dm_field_office_2_south" },
+      { "point": [ 1, 0, -1 ], "overmap": "dm_field_office_3_south" },
+      { "point": [ 0, 0, -1 ], "overmap": "dm_field_office_4_south" },
+      { "point": [ 1, 1, 1 ], "overmap": "dm_field_office_9_south" },
+      { "point": [ 0, 1, 1 ], "overmap": "dm_field_office_10_south" },
+      { "point": [ 1, 0, 1 ], "overmap": "dm_field_office_11_south" },
+      { "point": [ 0, 0, 1 ], "overmap": "dm_field_office_12_south" },
+      { "point": [ 1, 1, 2 ], "overmap": "dm_field_office_9_south" },
+      { "point": [ 0, 1, 2 ], "overmap": "dm_field_office_10_south" },
+      { "point": [ 1, 0, 2 ], "overmap": "dm_field_office_11_south" },
+      { "point": [ 0, 0, 2 ], "overmap": "dm_field_office_12_south" },
+      { "point": [ 1, 1, 3 ], "overmap": "dm_field_office_13_south" },
+      { "point": [ 0, 1, 3 ], "overmap": "dm_field_office_14_south" },
+      { "point": [ 1, 0, 3 ], "overmap": "dm_field_office_15_south" },
+      { "point": [ 0, 0, 3 ], "overmap": "dm_field_office_16_south" },
+      { "point": [ 1, 1, 4 ], "overmap": "dm_field_office_17_south" },
+      { "point": [ 0, 1, 4 ], "overmap": "dm_field_office_18_south" },
+      { "point": [ 1, 0, 4 ], "overmap": "dm_field_office_19_south" },
+      { "point": [ 0, 0, 4 ], "overmap": "dm_field_office_20_south" }
+    ],
+    "locations": [ "land" ],
+    "city_distance": [ 40, -1 ],
+    "occurrences": [ 100, 100 ],
+    "flags": [ "DEFENSE_MODE", "GLOBALLY_UNIQUE" ]
+  },
+  {
+    "type": "mapgen",
+    "om_terrain": [ [ "dm_field_office_1", "dm_field_office_2" ], [ "dm_field_office_3", "dm_field_office_4" ] ],
```

Comment:
Are you using an older version of the Border Patrol Office mapgen?  I noticed the armory and contraband vault are missing.

Reply:
It would seem I'm using an older version. I made this on the 09/02/2023 release before I updated.

Reply:
Yep that makes sense #68012 was merged 2 weeks ago.   :)

Reply:
Okay. I'll have to update it. Any idea as to why the code crashes when run?